impact.py

```diff
@@ -114,7 +114,7 @@ def compute(self,
             baseline=0,
             bounds=None,
             clip_flat_curve=True,
-            t_star=None):
+            t_star_path=None):
```

Comment:
why can't we provide t_star here? What if we have tstar instead of a path?

Reply:
I am not sure how we would know if t_star has already been precomputed. Right now, we only know because we read a file path. If its empty, we compute t_star. How else would you imagine getting t_star? It's not clear to me how to know whether or not we've computed t_star for a given set of covariates, iams etc.